src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Figure implements SkillOfDrawing, SkillOfCalculation {
+    protected String color;

```

Comment:
You created enum color, but you use `String` for color in you `Figure` class. Why?